python/src/acp_sdk/models/models.py

```diff
@@ -117,6 +117,33 @@ class CitationMetadata(BaseModel):
     description: Optional[str]
 
 
+class TrajectoryMetadata(BaseModel):
+    """
+    Represents trajectory information for an agent's reasoning or tool execution
```

Comment:
I'd refelect the changes that i proposed in documentation here as well.

Reply:
Same - consistency of terminology would be good.